common/payment/PayBC Sequence.md

```diff
@@ -2,32 +2,33 @@
 
 ```mermaid
 sequenceDiagram
-    participant Browser
-    participant onRouteBC
-    participant PayBC
-    participant CFS
     Browser->>onRouteBC: Request to pay for permit
-    onRouteBC->>PayBC: Generate token
-    note over onRouteBC,PayBC: Supply client id and secret
-    PayBC-->>onRouteBC: Send bearer token
     onRouteBC->>onRouteBC: Generate transaction number
     onRouteBC->>onRouteBC: Save transaction number to DB
-    onRouteBC->>onRouteBC: Set permit status to 'WAITING FOR PAYMENT'
+    onRouteBC->>onRouteBC: Set permit status to 'WAITING_PAYMENT'
     onRouteBC-->>Browser: Send redirect to PayBC
-    Browser->>PayBC: Forward to payment page
-    note over Browser,PayBC: Include all transaction details such as<br/>transaction number, amount, and auth token
-    PayBC-->>Browser: Display payment details form
-    Browser->>PayBC: Supply payment information (credit card)
-    PayBC->>PayBC: Process payment
-    note over PayBC: Assume for this diagram<br/>that the payment is<br/>successful
-    PayBC-->>Browser: Send redirect with details of transaction
+    Browser->>PayBC: Forward to PayBC
+    note over Browser,PayBC: Include all transaction details such as<br/>transaction number, amount, hash, etc.
```

Comment:
is the `etc` important to itemize?

Reply:
The transaction details sent over to PayBC are covered in the Postman collection that's been updated alongside this. Good note.